src/imageio/format/jpeg.c

```diff
@@ -489,6 +563,7 @@ void gui_reset(dt_imageio_module_format_t *self)
 {
   dt_imageio_jpeg_gui_data_t *g = (dt_imageio_jpeg_gui_data_t *)self->gui_data;
   dt_bauhaus_slider_set(g->quality, dt_confgen_get_int("plugins/imageio/format/jpeg/quality", DT_DEFAULT));
+  dt_bauhaus_combobox_set(g->subsample, 0);    // auto
```

Comment:
Use DT_SUBSAMPLE_AUTO instead of 0 here.

Reply:
Changed.